### DISCUSSION TITLE
Have I lost my account?

### DISCUSSION THREAD

**Discussion by marcofiletto** [community/community#61753] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi! 
I think I have made a mistake.
A replaced my phone and when I tried to access my GitHub account it asked me for a 2FA that I did not have because I had not my phone anymore...
I checked for the other alternatives but none of them works for me (someone the saved key on a file that I could not find)...
So I created another account with the same email and if I understood correctly, this 'replace' my old account...
The thing is, I would like to recover my old account, I have a lot of repositories and info that I need.

Is there something I can do?
thanks,

**Comment by w8ste** [comment]
You can search for your old account via user search. If the account is still there you could at least get your public repositories back. If you can find, you could try to contact support. You can also try the following from the GitHub docs: [this](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

**Reply by marcofiletto** [reply]
Yep, the user still exists... but I'm more interested in the private repositories.

**Reply by w8ste** [reply]
Well, if the steps from the link above won't work I think support is your best option :)

**Reply by marcofiletto** [reply]
Does Github have personal contact support? because all links redirect me to this forum

**Reply by w8ste** [reply]
Read [this](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket#creating-a-support-ticket)
and create a ticket [here](https://support.github.com/request)
